### ISSUE TITLE
What's going on?

### ISSUE THREAD

**Issue by jacklollz2** [EnergizedProtection/block#980]
Numerous problems with Energized Regional Extension.
Youtube, Parsec, Guilded, etc. all down.
Did you guys pull the plug?

**Comment by Lichtenshtein** [comment]
Some of critical domains newly blocked by Energized Regional Extension list:

www.google.com
www.youtube.com
feeds.feedburner.com
avatars.githubusercontent.com
www.gstatic.com
connectivitycheck.gstatic.com

**Comment by antplus** [comment]
Clearly given up, blocking common domains and even NextDNS.  Removed the Energized lists for now, will revisit this thread in the future to see if it's sorted.

**Comment by jarekt** [comment]
steamcommunity.com also down, why?

**Comment by nightznero** [comment]
Imagine u block google github yt fb from one list that is not about extreme privacy xDDD idk how bad Energized.Pro lists going worse and worse by time. Never seen more problematic list like Energized

**Comment by Wizardling** [comment]
I'm out. This is now unusable for me. Who but the most extreme privacy zealots will put up with so many useful sites being outright blocked? Urrgh.

**Comment by itsdoublearon** [comment]
This has gotten absolutely ridiculous now. Way too aggressive in blocking now. Not enabling this list anymore.

**Comment by eisklwksjs** [comment]
Energized lists are empty, maybe abandoned or fell prey to a stupid hacker😞

**Comment by nightznero** [comment]
Like 3h ago everything has been fixed and right now again. Or start looking what lists are u adding or u have rat in repository cause idk how have u fixed it for period of time and again fvcked up list

**Comment by luxzg** [comment]
More than a day, on and off, my main concerns have een (and again are) google.com and nextdns.io. I couldn't search, I couldn't check logs. Had to disable Private DNS to go to NextDNS, to see that Energized is acting up. I'm subscribing to this issue just in case, but My Energized Basic + Regional is gone until there's been a week without issues.

**Comment by vladmiller** [comment]
Yeah, it blocked linkedin, google domains, even nextdns.io... crazy. Unsubscribed as well.

**Comment by RouterCFW** [comment]
Facebook.com is now blocked as well

**Comment by file3p1rat** [comment]
reddit is blocked in ultimate

**Comment by S7venLights** [comment]
Adding to the list of things blocked in the past 2 hours:
netflix.com
assets.pinterest.com
LinkedIn

**Comment by RouterCFW** [comment]
Yeah just ditch Energized it’s an abandoned list

**Comment by Taubin** [comment]
Does anyone have any alternatives? This is unacceptable.

**Comment by asdfzxcvbn** [comment]
well right now i'm using a huge amount of lists, but some good ones are oisd, 1hosts, notracking, and lightswitch05's ads and tracking.

**Comment by xaftu** [comment]
blocked reddit. based?

**Comment by S7venLights** [comment]
> Yeah just ditch Energized it’s an abandoned list

It can't be abandoned, it was updated an 3 hours ago, must be a mistake.

**Comment by nightznero** [comment]
OISD
GoodbyeAds(include lightswitch05)
and 1Hosts that has been included in Energized.Pro,
it was main reason why i was usuing it but guess its time to switch from Energized just to clear 1Hosts

**Comment by rydhoms** [comment]
I'll have to remove all energized lists for now until there is more information on this issue, really can't do anything important with many important domains website and domain app service blacklisted.

**Comment by RouterCFW** [comment]
> > Yeah just ditch Energized it’s an abandoned list
> 
> It can't be abandoned, it was updated an 3 hours ago, must be a mistake.

It’s a bot that update it

**Comment by luxzg** [comment]
> Does anyone have any alternatives? This is unacceptable.

This is my current statistics from NextDNS.io :

```
List / Number of queries blocked
Energized Basic 55,744
Goodbye Ads 41,772
1Hosts (Lite) 35,840
Lightswitch05 - Ads & Tracking 21,370
oisd 20,812
AdGuard DNS filter 20,656
```
Obviously Energized is gone now :p 5 others are all well know and much loved. I haven't had issues in past several months since I added some extras. Well, until Energized went bananas. Only thing that may annoy some people is Goodbye Ads blocking sponsored links, so eg top Google sponsored results / links won't work, but those are ads... May be useful... So gray area. Discard that list if annoyed by that. Rest are safe for general population, including any social apps, websites, and such

**Comment by yegors** [comment]
As of ~6hrs ago, the "ultimate" list has no domains in it.

**Comment by mad-it** [comment]
This started a few days ago... please have a look at https://github.com/EnergizedProtection/block/issues/973 🙈

**Comment by botthebestplayerintheworld** [comment]
https://github.com/nextdns/metadata/issues/1145
https://github.com/EnergizedProtection/block/issues/971
https://github.com/EnergizedProtection/block/issues/972
https://github.com/EnergizedProtection/block/issues/973
https://github.com/EnergizedProtection/block/issues/974
https://github.com/EnergizedProtection/block/issues/975
https://github.com/EnergizedProtection/block/issues/976
https://help.nextdns.io/t/m1hs207/energized-ultimate-lists-blocking-nextdns
https://help.nextdns.io/t/q6hs204/nextdns-io-addresses-added-to-a-blocklist-internet-goes-down
https://github.com/EnergizedProtection/block/issues/977
https://help.nextdns.io/t/83hsb6l/energized-ultimate-have-0-entries
yes

**Comment by botthebestplayerintheworld** [comment]
https://github.com/yokoffing/NextDNS-Config/issues/10
https://github.com/nextdns/metadata/pull/1148
Not using Energized anymore

**Comment by Garbg** [comment]
Blocklist is empty on AGH and site is empty and every link leads to 404

**Comment by asdfzxcvbn** [comment]
it looks like their homepage and README are back, maybe the maintainers realized?

**Comment by Garbg** [comment]
Either they got hacked, or they merged pull requests without reviewing them

**Comment by yegors** [comment]
All lists are blank now. Well, it was a good run boys. Time to abandon ship. I recommend OISD list.

**Comment by Taubin** [comment]
> it looks like their homepage and README are back, maybe the maintainers realized?

At this point, I have zero faith that the maintainers have any idea what they are doing.

**Comment by marcoclg19** [comment]
> All lists are blank now. Well, it was a good run boys. Time to abandon ship. I recommend OISD list and shilling for my own DNS service: https://controld.com (our lists are bespoke).

Can I make a suggestion? Can you remove the Energized lists and add those of  @hagezi?

**Comment by yegors** [comment]
@RouterCFW We have a CLI utility being released shortly (this week), which runs on routers. 

@marcoclg19 It was removed. We'll look into the list you suggested, however out bespoke Ads & Trackers list is really good. Many people who wanted 3rd party lists switched back to our own due to amounts of false positives found in community lists. Give it a shot :)

**Comment by RouterCFW** [comment]
> @RouterCFW We have a CLI utility being released shortly (this week), which runs on routers.
> 
> @marcoclg19 It was removed. We'll look into the list you suggested, however out bespoke Ads & Trackers list is really good. Many people who wanted 3rd party lists switched back to our own due to amounts of false positives found in community lists. Give it a shot :)

Wait what 😳❤️

**Comment by yegors** [comment]
Not public yet, but will be published here: https://github.com/Control-D-Inc/

**Comment by asdfzxcvbn** [comment]
no, doesn't look like it. what worries me even more is that the automatic updating of the lists has stopped. the last commit to this repo was 4 days ago, while it usually updates every ~6 hours. i've just sent an email to a maintainer and will share it here if i get a response.

**Comment by Joomsy** [comment]
This feels like a compromise to me, or a disgruntled dev. Any and all social media, and my banking app got blocked. Now I'm seeing others here reporting things like Google being blocked. I only used the regional and basic list, but the regional one is what pretty umbrella blocked everything. I only noticed Basic blocking Imgur, but I don't really use social media beyond image hosts and YouTube, so there's likely others. Really feels like somebody maliciously did this, rather than incompetently.

@yegors Thanks for offering an alternative. As fun as it would be to amasse several other lists, I much prefer a centralized solution. Will keep an eye on your repo for the public release.

**Comment by yegors** [comment]
I spoke to one of the devs on Telegram, he said "I am just a little busy with life. Hope to get it fixed by next week."

Take this as you will.

**Comment by RouterCFW** [comment]
> @RouterCFW We have a CLI utility being released shortly (this week), which runs on routers.
> 
> @marcoclg19 It was removed. We'll look into the list you suggested, however out bespoke Ads & Trackers list is really good. Many people who wanted 3rd party lists switched back to our own due to amounts of false positives found in community lists. Give it a shot :)

Will it forward local device names and mac etc so you can filter them in logs ?

**Comment by Taubin** [comment]
> I spoke to one of the devs on Telegram, he said "I am just a little busy with life. Hope to get it fixed by next week."
> 
> Take this as you will.

It's a bit too late for me and I'm sure many others. Something like this they simply cannot let go until "next week". It's already been going for over a week now.

**Comment by yegors** [comment]
@RouterCFW Email me (yegor@controld.com), I'll send you a preview 1.0 build to play around with (you can do what you asked, but not as "magically" as with NextDNS). Don't wanna keep hijacking this thread more than I already have.

**Comment by asdfzxcvbn** [comment]
according to yegor's quote, a dev said they had been planning on fixing it within a week. that was three weeks ago, and only one commit has been made since then. EnergizedProtection/block@31bbf93 is the only commit that's been made recently, and it only updates the readme to add a sponsors section. however, the commit message does say this:

> *Hope to bring back the service up soon.

maybe this means something? well, even if it does, i dont know if it would be a good idea to retrust the devs who "magically" disappeared for weeks and couldnt even make an announcement, only to magically return again. it's none of my business whether you decide to use the list again, so do what you want.

**Comment by asdfzxcvbn** [comment]
it looks like their [homepage](https://energized.pro) has been updated too. now it just reads:

> We will be back soon. 

instead of an empty page. they might make an announcement sooner or later.

**Comment by Wizardling** [comment]
Trusting this blocklist again after such a massive and sustained screwup/sabotage, is a fool's errand IMO. Especially when there are a number of high quality alternatives. It is well past time to move on.

**Comment by silverwings15** [comment]
> Trusting this blocklist again after such a massive and sustained screwup/sabotage, is a fool's errand IMO. Especially when there are a number of high quality alternatives. It is well past time to move on.

been very much out of the loop, can you please provide some such alternatives?

**Comment by Wizardling** [comment]
> > Trusting this blocklist again after such a massive and sustained screwup/sabotage, is a fool's errand IMO. Especially when there are a number of high quality alternatives. It is well past time to move on.
> 
> been very much out of the loop, can you please provide some such alternatives?

I use several alternative blocklists. The main ones are:

https://oisd.nl/
https://pgl.yoyo.org/adservers/
https://someonewhocares.org/hosts/
https://fanboy.co.nz/

I hope that helps you out :-)

**Comment by silverwings15** [comment]
thank you for your suggestion guys, the StevenBlack and OISD lists look great

**Comment by earthsound** [comment]
> > Trusting this blocklist again after such a massive and sustained screwup/sabotage, is a fool's errand IMO. Especially when there are a number of high quality alternatives. It is well past time to move on.
> 
> been very much out of the loop, can you please provide some such alternatives?

hazezi's DNS Blocklists is a great, maintained collection:
https://github.com/hagezi/dns-blocklists

**Comment by marcoclg19** [comment]
> @marcoclg19 It was removed. We'll look into the list you suggested, however out bespoke Ads & Trackers list is really good. Many people who wanted 3rd party lists switched back to our own due to amounts of false positives found in community lists. Give it a shot :)

So... any chance for Hagezi blocklists? @yegors :)

However, [this page](https://kb.controld.com/en/3rd-party-filters) should also be updated.

**Comment by yegors** [comment]
@marcoclg19 Yes, in fact this is available right now. Check the updated 3rd party filters page for resolver details :) 

https://kb.controld.com/en/3rd-party-filters

**Comment by hagezi** [comment]
@yegors Thank you for including my lists. If there is anything I can do, just let me know.

**Comment by trollybusman** [comment]
Control D is awesome! Glad to see this thread (not the fact that energized is EOL now).

**Comment by silverwings15** [comment]
> hazezi's DNS Blocklists is a great, maintained collection: https://github.com/hagezi/dns-blocklists

seemingly a great all-in-one list, but due to it being a DNS list, basic cosmetics filters like `reddit.com##.promotedlink` are not available, forcing me to reenable EasyList in my uBO after encountering ads on reddit

**Comment by t3dium** [comment]
> > > Trusting this blocklist again after such a massive and sustained screwup/sabotage, is a fool's errand IMO. Especially when there are a number of high quality alternatives. It is well past time to move on.
> > 
> > 
> > been very much out of the loop, can you please provide some such alternatives?
> 
> hazezi's DNS Blocklists is a great, maintained collection: https://github.com/hagezi/dns-blocklists

+1 for hagezi's lists, haven't ran into a single false positive in a month of usage, and thats on the 2.1 mill domain list.

**Comment by S7venLights** [comment]
https://filterlists.com/
